File: app/src/main/java/org/exallium/tradetracker/app/model/entities/LineItem.java
package org.exallium.tradetracker.app.model.entities;

import java.util.Date;

public class LineItem extends Record<LineItem> {
    public String description;
    public Card card;
    public long value;
    public Trade trade;
    public Date lastUpdated;
    public boolean direction;
}

File: app/src/main/kotlin/org/exallium/tradetracker/app/model/entities/LineItem.kt
package org.exallium.tradetracker.app.model.entities

import org.joda.time.LocalDate
import java.util.Date

public class LineItem : Record<LineItem>() {
    public var description: String? = null
    public var card: Card? = null
    public var value: Long = 0
    public var trade: Trade? = null
    public var lastUpdated: Date = LocalDate.now().toDate()
    public var direction: Boolean = false
}
